source/developer/fx-guidelines.rst

```diff
@@ -61,15 +61,27 @@ That said, forgiving does not mean allowing the product to be used incorrectly b
 Technical Guidelines
 ********************
 
-Bootstrap Classes
```

Comment:
If bootstrap classes are still recommended, I'll add it back. I think we've moved on from them though?